DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentDataServiceTests.cs

```diff
@@ -0,0 +1,466 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
```

Comment:
I assume this has just been renamed from DiagnosticAssessmentServiceTests and the content hasn't changed? 🙂

Reply:
Correct, just a file rename